.tidyallrc

```diff
@@ -0,0 +1,3 @@
+[PerlTidy]
+select = **/*.{pl,pm,t} tools/tidyall tools/perlcritic tools/update-deps
```

Comment:
So why doesn't a symlink to '.tidyallrc' work? Is this the first repo where we want to share this file?

Neither [os-autoinst](https://github.com/os-autoinst/os-autoinst/blob/master/.tidyallrc) nor [openQA](https://github.com/os-autoinst/openQA/blob/master/.tidyallrc) uses a link, so maybe it never worked 🤔

Reply:
I think a copy is fine as the files to tidy or to ignore might differ.
Also, the tools/* are all symlinks to the common repo, right? So we don't need to tidy them, as they come from the external repo